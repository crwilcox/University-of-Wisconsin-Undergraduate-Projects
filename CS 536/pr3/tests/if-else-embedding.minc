bool cond1 = false;




bool cond2 = true;
int x = 0;
if (cond1) 





if (cond2) x=3; 
else x=6; 
else x=5;
