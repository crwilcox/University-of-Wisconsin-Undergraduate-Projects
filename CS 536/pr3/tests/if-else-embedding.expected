bool cond1 = false;
bool cond2 = true;
int x = 0;
